dist/dialog/ds4/dialog.css

```diff
@@ -162,11 +161,24 @@
   -webkit-transform: translateX(0);
           transform: translateX(0);
 }
-@media (min-width: 768px) {
+@media (max-width: 600px) {
   .dialog__window:not(.dialog__window--full) {
-    width: calc(50% - 14px);
-    min-width: calc(50% - 14px);
-    max-width: calc(88% - 15px);
+    width: calc(100% - 16px);
+    min-width: calc(100% - 16px);
+    max-width: calc(100% - 16px);
+  }
+}
+@media (min-width: 601px) and (max-width: 768px) {
+  .dialog__window:not(.dialog__window--full) {
+    width: calc(88% - 16px);
+    min-width: calc(88% - 16px);
+    max-width: calc(88% - 16px);
+  }
+}
+@media (min-width: 769px) and (max-width: 1024px) {
+  .dialog__window:not(.dialog__window--full) {
+    width: calc(75% - 32px);
+    max-width: 616px;
```

Comment:
i think this is not needed since its set default(L18)

Reply:
Yes, it’s needed. Without it the higher order selector overrides the width settings and it becomes too wide.

Please try and remove it and test it to confirm.

Reply:
and, all the desktop dialog would be small compare to the big screens since we its restricted to max-width 616px. I think 616px on bigger screen would look like small modal. max-width can be set in percentage of viewport. thought?
 and, pages support following breakpoints,
@breakpoint-smallest: 480px;
@breakpoint-smaller:  600px;
@breakpoint-small:    768px;
@breakpoint-medium:   960px;
@breakpoint-large:    1024px;
@breakpoint-larger:   1280px;
@breakpoint-wide-ads: 1350px;
@breakpoint-largest:  1420px;

Reply:
Yeah, and those current breakpoints don't apply to a dialog. It has it's own breakpoints and own set of rules.